roles/keystone/templates/etc/serverspec/keystone_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+[['keystone.conf', 640], ['keystone-paste.ini', 640], ['policy.json', 644]].each do |file|
+  describe file("/etc/keystone/#{file[0]}") do
+    it { should be_mode file[1] }
+    it { should be_owned_by 'keystone' }
+    it { should be_grouped_into 'keystone' }
+  end
+end
+
+describe file('/var/log/keystone/keystone-all.log') do
+  it { should be_mode 644 }
+  it { should be_owned_by 'keystone' }
+  it { should be_grouped_into 'adm' }
+end
+
+
+describe file('/var/log/keystone/keystone-manage.log') do
+  it { should be_mode 644 }
+  it { should be_owned_by 'root' }
+  it { should be_grouped_into 'adm' }
+end
+
+describe file('/etc/logrotate.d/keystone') do
+  file_contents = [ '# Generated by Ansible.',
+  		     '# Local modifications will be overwritten.',
+		     '',
+		     '/var/log/keystone/*.log',
+		     '{',
+		     '  daily',
+		     '  missingok',
+		     '  rotate 7',
+		     '  compress',
+		     '}'].join("\n")
+
+  it { should contain file_contents }
+end
+
+describe file('/etc/keystone/keystone.conf') do
+  file_contents = [ '[ssl]',
+  		    'enable = False'].join("\n")
+  it { should contain file_contents }
+  it { should contain 'debug = True' }
```

Comment:
Whether debug or verbose is true or not would depend on whether the settings for those are set

https://github.com/blueboxgroup/ursula/blob/master/roles/keystone/templates/etc/keystone/keystone.conf#L2